storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1020,6 +1022,13 @@ struct my_option xb_client_options[] = {
      (uchar *)&opt_force_non_empty_dirs, (uchar *)&opt_force_non_empty_dirs, 0,
      GET_BOOL, NO_ARG, 0, 0, 0, 0, 0, 0},
 
+    {"no-server-version-check", OPT_NO_SERVER_VERSION_CHECK,
+     "This option allow backup to proceed"
```

Comment:
```suggestion
     "This option allows backup to proceed"
```

Reply:
Why do I still see "allow" instead of "allows?" I see that you marked it as resolved..